cmd/internal/shared/devicetemplateconverter/config.go

```diff
@@ -19,4 +19,6 @@ import (
 )
 
 // DefaultDeviceTemplateConverterConfig is the default configuration for the Device Template Converter.
-var DefaultDeviceTemplateConverterConfig = devicetemplateconverter.Config{}
+var DefaultDeviceTemplateConverterConfig = devicetemplateconverter.Config{
+	Enabled: []string{"the-things-stack"},
```

Comment:
I think it should always be enabled, regardless of configuration here. I think this is a special case.

In existing setup that set the set of enabled converters, this field gets overridden, and `the-things-stack` should be enabled explicitly.

Reply:
Consideration here is that we should always be able to convert from our own template file.